MapboxDirections/MBRoute.swift

```diff
@@ -174,7 +174,7 @@ open class Route: NSObject, NSSecureCoding {
      
      The route options object’s profileIdentifier property reflects the primary mode of transportation used for the route. Individual steps along the route might use different modes of transportation as necessary.
      */
-    @objc open let routeOptions: RouteOptions
+    @objc open let routeOptions: DirectionOptions
```

Comment:
This is an issue. What will happen in the navigation SDK when we use this param to reroute? Not all properties will be present.

Reply:
If we have RouteOptions and MatchOptions be siblings under DirectionsOptions, then Route and Match must be siblings under DirectionsResult (or whatever we’d call it). Then Route and Match would have a covariant `options` property that has the appropriate type.